src/redux/actions/cats.js

```diff
@@ -1 +1,2 @@
 export const onCat = () => ({ type: 'cats/RANDOM_CAT' })
+export const adoptCat = () => ({ type: 'cats/ADOPT_CAT' })
```

Comment:
What we need to do here is pass some data (eg. the cat that we are adopting) with this action. Check out our [Redux video](https://www.youtube.com/watch?v=gxWau7QSJfA&index=7&list=PL7ahsr4elhyCoQhDtBuumDmEkVQ_-0qlj&t=0s) which tells you how to send data when firing an action